go/oasis-node/cmd/debug/txsource/transferworkload.go

```diff
@@ -0,0 +1,129 @@
+package txsource
```

Comment:
Can you put the workloads into a `workload` directory (e.g., this one would be `transfer.go`)?

Reply:
ok, as long as I export a bunch of stuff

Reply:
oh wait, not much stuff